design-doc-template.adoc

```diff
@@ -0,0 +1,39 @@
+= Title
+:author:            Your Name
+:email:             your.email@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
```

Comment:
In Jeff's example, which I copied, the JIRAs were listed up here in the metadata.  I guess that is not the right thing to do?

Reply:
This could really be deleted. I just thought *maybe* it would be useful to reference the issue like `{issue-base-url}/WCORE-XXX`. It might not really be worth it though.

I did base this mostly off https://developer.jboss.org/wiki/DesignNotesForGracefulShutdownUponReceiptOfASIGTERM.

Reply:
+1 to removing this.